packages/auth/src/signature/middleware.test.ts

```diff
@@ -408,7 +406,13 @@ describe('Signature Service', (): void => {
     })
 
     test('middleware succeeds if BYPASS_SIGNATURE_VALIDATION is true with bad signature', async (): Promise<void> => {
-      config.bypassSignatureValidation = true
+      const altDeps = await initIocContainer({
+        ...Config,
+        bypassSignatureValidation: true
```

Comment:
And creating a new container can happen in tests like this so we don't have to modify `Config`